src/mgmt/storage_query.pb-c.h

```diff
@@ -236,6 +238,48 @@ struct  _Mgmt__DevStateResp
     , 0, (char *)protobuf_c_empty_string, (char *)protobuf_c_empty_string }
 
 
+struct  _Mgmt__DevReplaceReq
+{
+  ProtobufCMessage base;
+  /*
+   * UUID of old (hot-removed) blobstore/device
+   */
+  char *old_dev_uuid;
+  /*
+   * UUID of new (hot-plugged) blobstore/device
+   */
+  char *new_dev_uuid;
+};
+#define MGMT__DEV_REPLACE_REQ__INIT \
+ { PROTOBUF_C_MESSAGE_INIT (&mgmt__dev_replace_req__descriptor) \
+    , (char *)protobuf_c_empty_string, (char *)protobuf_c_empty_string }
+
+
+struct  _Mgmt__DevReplaceResp
+{
+  ProtobufCMessage base;
+  /*
+   * DAOS error code
+   */
+  int32_t status;
+  /*
+   * UUID of old (hot-removed) blobstore/device
+   */
+  char *old_dev_uuid;
+  /*
+   * UUID of new (hot-plugged) blobstore/device
+   */
+  char *new_dev_uuid;
+  /*
+   * BIO device state
+   */
+  char *dev_state;
```

Comment:
I don't quite see why dev_replace reply has "old_dev_uuid", "new_dev_uuid" and "dev_state"? It looks to me a status is enough.

Reply:
I guess "old_dev_uuid" is not necessary, however the new device uuid and also the state are printed to the console at the end of the command, so that's why they are apart of the reply

Reply:
I just think that both old and new UUID is known by client, so looks it's not necessary to carry them back from server by request reply.